docs/install.md

```diff
@@ -2,11 +2,13 @@
 
 # Installation
 
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc), and on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). This page describes how to install Nextflow.
```

Comment:
Think it would be good to link to the updating page from here, or at the bottom?